test/test_noise.py

```diff
@@ -65,6 +65,11 @@ def test_noise_psd(self):
         ave = ratio.numpy().mean()
         self.assertAlmostEqual(ave, 1, 1)
 
+    def test_noise_reproducible(self):
+        ts1 = normal(20, 30, sample_rate=16384, seed=87693)
+        ts2 = normal(25, 35, sample_rate=16384, seed=87693)
+        self.assertEqual(ts1.time_slice(25, 30), ts2.time_slice(25, 30))
```

Comment:
@GarethDaviesGW Does assert equal here really know how to compare time series? (also there may be slight numerical error, so an 'almost equal' may be more appropriate. You may need to take the diff and check that the maximum difference is below some reasonable numerical threshold.

Reply:
assertequal was working here, I had used ts1(...) == t2(...), but the "False is not True" error output wasn't helpful

Reply:
if it fails, the assertequal gives:
```
AssertionError: <pycbc.types.timeseries.TimeSeries object at 0x7fc563788790> != <pycbc.types.timeseries.TimeSeries object at 0x7fc5637880d0>
```

Which is (slightly) more descriptive!